### DISCUSSION TITLE
Option to pin chat conversations

### DISCUSSION THREAD

**Discussion by jcdiv47** [danny-avila/LibreChat#2610] [Feature Requests & Suggestions]
I was wondering if many users(besides me) could benefit from the addition of the ability to pin chat conversations. One of the use cases for me is I would set up a chat conversation with a specific preset for a certain topic, e.g., an assistant that provides answers to a specific area. I would go back to the same chat session from time to time with similar questions/requests.

For example, I may define a preset that serves as an assistant that answers all my questions regarding bash commands. It would be more convenient if I could pin the chat and easily go back to it. For now my solution is simply search for keywords every time, assuming a properly named title.

<img width="255" alt="Screenshot 2024-05-04 at 00 08 01" src="https://github.com/danny-avila/LibreChat/assets/45190310/547baba8-5b9d-41c9-9fa6-e998d8c59eb5">

**Comment by danny-avila** [comment]
Great idea, could work similar to how ChatGPT pins assistants.. but I do prefer conversations being pinned in LibreChat as that is more universal.

**Reply by jcdiv47** [reply]
Yes pinning conversations is exactly what I want, does this feature already exist?

**Comment by henricook** [comment]
I'd love that feature too, and I've opened a PR for it here: https://github.com/danny-avila/LibreChat/pull/7550

**Reply by henricook** [reply]
I'm afraid I've had to close this PR because of no engagement from maintainers after 6+ months. If you have a maintainer's ear and they're ready to review/merge it please get in touch and I can reopen it.
